src/common/span.rs

```diff
@@ -106,7 +115,9 @@ impl Span {
     /// so if the `Span` is along an invalid byte boundary or
     /// is empty, the program will panic.
     pub fn contents(&self) -> String {
-        if self.is_empty() { panic!("An empty span does not have any contents") }
+        if self.is_empty() {
+            panic!("An empty span does not have any contents")
+        }
```

Comment:
I'm not a huge fan of breaking single expressions in blocks across multiple lines unless it really improves readability. I know readability is subjective; I think the amount of space source code occupies should be correlated with how difficult that code is to understand.